src/constants/constants.js

```diff
@@ -264,7 +280,6 @@ export const OPERATION_ID_STATUS = {
 export const OPERATIONS = {
     PUBLISH: 'publish',
     GET: 'get',
```

Comment:
Should we have here query aslo?

Reply:
Maybe yet, but then we also need to create tables for it. I'm not sure why this wasn't done before but it might be that by "operations" we only considered network operations